src/client/pages/WelcomeSocial.tsx

```diff
@@ -91,6 +94,7 @@ export const WelcomeSocial = ({
 					registrationFormSubmitOphanTracking(e.target as HTMLFormElement);
 					return undefined;
 				}}
+				additionalTerms="Newsletters may contain info about charities, online ads, and content funded by outside parties."
```

Comment:
Is this really meant to be 'info', not 'information'?

Reply:
it was meant to be information! Good spot.

I've also noticed it was used in 3 places. So following DRY, I added this commit [`5fc0be3` (#2640)](https://github.com/guardian/gateway/pull/2640/commits/5fc0be35b1be7a9a48c6e1a0587d1f9067177317) which moves everything into an importable string const.